src/navigator/DrawerNavigator.tsx

```diff
@@ -283,7 +285,20 @@ export default function DrawerNavigator() {
           name={Screens.BackupIntroduction}
           // @ts-expect-error component type in native-stack v6
           component={BackupIntroduction}
-          options={{ title: t('accountKey'), drawerIcon: AccountKey }}
+          options={{
+            drawerLabel:
+              !backupCompleted && getOnboardingExperimentParams().showBackupAlert
```

Comment:
I'm not sure how to solve this exactly, but I worry that this will log exposure events for all users viewing the side nav. I think we only wanted to consider users who saw the new screen in onboarding, though.

Reply:
lmk if you want to discuss solution ideas